client/src/components/UserPledges/style.css

```
@@ -0,0 +1,169 @@
+#root{
+  height:100vh;
+}
+.user-pledge{
+  width:100%;
+  height: 145px;
+  font-style: Raleway;
+  position: relative;
+  margin-top:18px;  
+}
+
+.user-pledge__image{
+  margin: auto;
+  width: 90%;
+  display: block;
+  height: 100%;
+  border-radius: 15px;
+}
+
+.user-pledge__title{
+  position: absolute;
+  bottom: 40%;
+  left: 7%;
+  font-weight: bold;
+  font-size: 24px;
+  color: #FFFFFF;
+  margin-block-start: 0;
+  margin-block-end: 0; 
+}
+
+.user-pledge__description{
+  width: 72%;
+  position: absolute;
+  bottom: 7%;
+  left: 7%;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 15px;
+  color: white;
+  word-wrap: break-word;
+}
+
+.user-pledge__enrollements{
+  position: absolute;
+  right: 3%;
+  top: 63%;
+  background: #F9BE02;
+  width:60px;
+  height: 41.38%;
+  color: #fff;
+  text-align: center;
+  border-radius: 50%;
+  border: 2px solid white;
+}
+
+.user-pledge__enrollements__image{
```

```suggestion
.user-pledge__enrollements-image{
```
we use the `__` just for the main container `user-pledge` and not for `enrollements`